rest-service/manager_rest/resource_manager.py

```diff
@@ -580,6 +583,37 @@ def delete_deployment(self,
         # Delete deployment data  DB (should only happen AFTER the workflow
         # finished successfully, hence the delete_db_mode flag)
         else:
+            # check for external targets
+            deployment_dependencies = self.sm.list(
+                models.InterDeploymentDependencies,
+                filters={'source_deployment': deployment})
+            external_targets = set(
+                json.dumps(x.external_target) for x in
```

Comment:
Let's change x to be informative, maybe "dependency"?

Reply:
1